### PR TITLE
change initial route to pingpong

### PR DESCRIPTION
**Pull request by jhroot**
- [ ] do not cache headers
- [ ] unit test

### CODE REVIEW

**Comment by thewilkybarkid** [comment]
Draft of https://github.com/libero/libero/issues/26 currently requires `Cache-Control: no-store, must-revalidate` and `Content-Type: text/plain; charset=UTF-8` headers.

We should add a first test too.

File: api/api.py [hunk 1]
```diff
@@ -2,6 +2,7 @@
 
 app = Flask(__name__)
 
-@app.route('/')
+
+@app.route("/ping")
 def hello():
```
Comment (thewilkybarkid) [inline api/api.py]: `ping()`?

Comment (jhroot) [reply api/api.py]: good point!

**Comment by jhroot** [comment]
re:test - yes, but this was just a minimal change to what was there to make sure I had the required permissions to interact with the project. (e.g. It also needs a comment for the function etc. )

**Comment by giorgiosironi** [comment]
Added checklist in place of description, this kind of small features are ideal for setting up things like `pytest` and other supporting things.

**Comment by jhroot** [comment]
As I commented above, I was checking the process for making a PR is working with an absolute minimal change. I will close this PR.

**Pull request closed without merging**